### PR TITLE
fix non-@ref tag in serialization (#25900)

### PR DESCRIPTION
**Pull request by shkoder**
ref #25900

### CODE REVIEW

File: akka-docs/src/main/paradox/serialization.md [hunk 1]
```diff
@@ -259,7 +259,7 @@ The recommended approach to do deep serialization of internal actor state is to
 ## Disabling the Java Serializer
 
 Since the `2.4.11` release of Akka it is possible to entirely disable the default Java Serialization mechanism.
-For compatibility reasons, the current (non-@ref:[Artery](remoting-artery.md)) remoting still uses Java
+For compatibility reasons, the current (@ref:[Artery](remoting-artery.md)) remoting still uses Java
```
Comment (patriknw) [inline akka-docs/src/main/paradox/serialization.md]: hm, I think this should be 
```
current (non-Artery) @ref:[Remoting](remoting.md) still uses Java
```

Comment (shkoder) [reply akka-docs/src/main/paradox/serialization.md]: indeed, addressed with 2ca437e

**Review by patriknw — APPROVED** [review]
LGTM, thanks

**Comment by patriknw** [comment]
OK TO TEST

**Review by raboof — APPROVED** [review]
This indeed makes more sense

**Pull request merged**